config/config/crds.yml

```diff
@@ -1306,6 +1309,9 @@ spec:
                   type: object
                 type: array
             type: object
+            x-kubernetes-validations:
+            - message: Expected service account or cluster, both cannot be specified
+              rule: 'has(self.serviceAccountName) ? !has(self.cluster) : has(self.cluster)'
```

Comment:
validation will be needed in "kind: PackageInstall" too, the next crd in this file.

Reply:
+1